eZ/Publish/SPI/Specification/Tests/Content/ContentContainerSpecificationTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Specification\Tests\Content;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\Core\Search\Tests\TestCase;
+use eZ\Publish\SPI\Specification\Content\ContentContainerSpecification;
+use eZ\Publish\SPI\Specification\Content\ContentSpecification;
+
+/**
+ * @covers \eZ\Publish\SPI\Specification\Content\ContentContainerSpecification
+ */
+final class ContentContainerSpecificationTest extends TestCase
+{
+    public function testConstructor(): void
+    {
+        $contentTypeSpecification = new ContentContainerSpecification();
+
+        $this->assertInstanceOf(ContentSpecification::class, $contentTypeSpecification);
+    }
+
+    /**
+     * @covers \eZ\Publish\SPI\Specification\Content\ContentContainerSpecification::isSatisfiedBy
+     * @dataProvider isSatisfiedByProvider
+     */
+    public function testIsSatisfiedBy(
+        bool $isContainer,
+        bool $shouldBeSatisfied
+    ): void
+    {
+        $contentContainerSpecification = new ContentContainerSpecification();
+
+        $contentTypeMock = $this->getMockBuilder(ContentType::class)
+            ->setConstructorArgs(
+                [['isContainer' => $isContainer]]
+            )
+            ->getMockForAbstractClass();
+
+        $contentMock = $this->createMock(Content::class);
+        $contentMock->expects($this->once())
+            ->method('getContentType')
+            ->willReturn($contentTypeMock);
+
+        $this->assertEquals(
+            $contentContainerSpecification->isSatisfiedBy($contentMock),
+            $shouldBeSatisfied
+        );
+    }
+
+    public static function isSatisfiedByProvider(): array
```

Comment:
Optional: when I see method starting with `is...` I immediately want to see `:bool` as a return type.
I'd avoid it, by e.g. naming it: `providerForIsSatisfiedBy` or, following verb based method naming conventions `getDataForIsSatisfiedBy`.
Both are fine for tests.
But not insisting.